pkg/meta/base_test.go

```diff
@@ -1999,9 +1999,14 @@ func testCheckAndRepair(t *testing.T, m Meta) {
 }
 
 func testDirStat(t *testing.T, m Meta) {
+	format := m.GetFormat()
+	format.DirStats = true
```

Comment:
it should be enabled in tests

Reply:
The `Init` method receives `format` as final state, we call `Init` for many times in tests and the `DirStat` is false by default. So if we want to ensure `DirStat` is enabled we should set it in each time we call `Init`.